operator/main.go

```diff
@@ -299,11 +304,29 @@ func setupKymaReconciler(
 	intervals controllers.RequeueIntervals,
 	options controller.Options,
 ) {
+	if flagVar.enableKcpWatcher {
+		fileInfo, err := os.Stat(flagVar.skrWatcherPath)
```

Comment:
This will always fail by default when starting locally since the path of the dockerfile and the path locally do not align. I would prefer if this can be fixed by using the same relative path in the dockerfile so the default happy path does not error out.

To run locally currently I have to run with
```
--enable-kcp-watcher --skr-watcher-path internal/charts/skr-webhook
```

Considering the path here. I do not understand why its in internal? Isnt this a pure go package?

Reply:
I have migrated the skr chart from the internal folder and aligned the relative path in the dockerfile to be the same as the default `--skr-watcher-path` flag value.